src/components/views/elements/Slider.tsx

```diff
@@ -83,18 +89,73 @@ export default class Slider extends React.Component<IProps> {
 
         let selection = null;
 
+        const onDrag = (e: React.DragEvent) => {
+            //  console.log(e.target.parentNode.parentNode)
+
+            const target = e.target as HTMLElement;
+            if (!target.parentNode) return null;
+            const parent = target.parentNode as HTMLElement;
+            if (!parent.parentNode) return null;
+            const slider = parent.parentNode as HTMLElement;
+            const rect = slider.getBoundingClientRect();
+
+            const x = e.clientX - rect.left; //x position within the element.
+            const width = rect.width;
+            const percent = x / width;
+            const value = Math.round(percent * (this.props.values.length - 1));
+            this.setState({ index: value, percent: percent * 100 });
+
+            //put the inner dot to the correct position
+            const innerDot = target.parentNode as any;
+            if (!innerDot) return null;
+            // innerDot.style.left = `${this.state.percent}%`;
+            const offset = this.offset(this.props.values, this.props.values[value]);
+            innerDot.style.left = `calc(-1.195em + " + ${offset} + "%)`;
```

Comment:
What does `-1.195em` relate to? How could we avoid that magic number?
If and we can't avoid it, we should add a thorough comment explaining how we got there